feature/experimental/isis/otg_tests/isis_metric_style_wide_not_enabled_test/isis_metric_style_wide_not_enabled_test.go

```diff
@@ -75,7 +75,6 @@ func configureISIS(t *testing.T, ts *session.TestSession) {
 
 	// Level configs.
 	level := isis.GetOrCreateLevel(2)
-	level.Enabled = ygot.Bool(true)
```

Comment:
This script uses session.go for base ISIS config and session.go has the deviation updated for level enabled. Additional config required for this specific testcase is being added in this test file apart from the config added in session.go. Since setting level enabled is already part of session.go, removing the duplicate level enabled config from this test file.